File: hospital-management/src/app/shared/custom-form.ts
import { QuestionBase } from "./question-base";

export class CustomForm {
  id: string;
  name: string;
  type: string;
  columns: number;
  alwaysInclude: boolean;
  fields: QuestionBase[];
  active: boolean;
  constructor() {
    this.id = this.name = this.type = "";
    this.columns = 0;
    this.alwaysInclude = this.active = false;
    this.fields = [];
  }
}

File: hospital-management-api/Models.MongoDB/CustomForm.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace hospital_management_api.Models.MongoDB
{
    public class CustomForm
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public bool Active { get; set; }
        //[BsonElement("name")]
        public string Name { get; set; }
        public string Type { get; set; }
        public int Columns { get; set; }
        public bool AlwaysInclude { get; set; }

        public List<Field> Fields { get; set; }
    }
}
